docs/guides/setup/setup_overview.md

```diff
@@ -0,0 +1,70 @@
+
+:::note Prerequisites
+- To best understand this document, you must have completed Step 1: Setup of the [Getting Started tutorial](../../tutorials/getting_started/initialize_a_data_context.md).
+:::
+	
+Getting started with Great Expectations is quick and easy.  Once you have completed your setup, you will have access to all of the features of Great Expectations from a single entry point: Your Data Context.  You will also have your Metadata Stores and Data Docs configured in the manner most suitable for your project's purposes.
+
+## The Setup process
+
+<!-- Brief outline of what the process entails.  -->
+
+Setup entails ensuring your system is prepared to run Great Expectations, installing Great Expectations itself, initializing your deployment, and doing some optional configuration of things like Data Docs and Datastores.  We'll look at each of these things in sequence.
+
+<!-- The following subsections should be repeated as necessary.  They should give a high level map of the things that need to be done or optionally can be done in this process, preferably in the order that they should be addressed (assuming there is one). If the process crosses multiple steps of the Universal Map, use the <SetupHeader> <ConnectHeader> <CreateHeader> and <ValidateHeader> tags to indicate which Universal Map step the subsections fall under. -->
+
+### 1. System Dependencies
+
+The first thing to take care of is making sure your work environment has the utilities you need to install and run Great Expectations.  These include a working Python install (version 3.8 or greater), the ability to pip install Python packages, an internet connection, and a browser so that you can use Jupyter notebooks.  Best practices are to use a virtual environment for your project's workspace.
+
+If you are having trouble with any of these, our documentation on [Supporting Resources](../../reference/supporting_resources.md) will direct you to more information and helpful tutorials.
+
+## 2. Installation
+
+Installing Great Expectations is a simple pip command.  From the terminal, execute:
+
+```markup title="Terminal command:"
+pip install great_expectations
+```
+
+Running this command in an environment configured to accept Python pip install commands will handle the entire installation process for Great Expectations and its dependencies.
+
+## 3. Initialize a Data Context
+
+Your Data Context contains the entirety of your Great Expectations project and provides the entry point for all of the primary methods you will use to configure and interact with Great Expectations.  That's why the first thing you'll do once you've installed Great Expectations will be to initialize your Data Context.  And this is another one-line command.  Simply go to the root folder for your project and execute:
```

Comment:
Another style guide question: "Project" or "Deployment"?

Either is fine. I lean towards "deployment," but it's not my decision. Let's pick one and standardize on it.

Reply:
```suggestion
Your Data Context contains the entirety of your Great Expectations project and provides the entry point for all of the primary methods you will use to configure and interact with Great Expectations.  That's why the first thing you'll do once you've installed Great Expectations will be to initialize your Data Context.  This is another one-line command.  Simply go to the root folder for your project and execute:
```